nginx-stable/snapcraft.yaml

```diff
@@ -0,0 +1,63 @@
+name: nginx
+version: "1.11.1"
+summary: nginx is an HTTP and reverse proxy server
+description: |
+  nginx ("engine x") is an HTTP and reverse proxy server, a mail proxy server,
+  and a generic TCP/UDP proxy server, originally written by Igor Sysoev.  For a
+  long time, it has been running on many heavily loaded Russian sites including
+  Yandex, Mail.Ru, VK, and Rambler.
+confinement: devmode
+
+apps:
+  nginx:
+    command: usr/nginx/nginx
+    stop-command: nginx -s quit
+    daemon: simple
+    plugs: [network, network-bind]
+
+parts:
+  nginx:
+    source: http://hg.nginx.org/nginx/archive/default.tar.gz
+    plugin: autotools_nginx
+    configflags:
+      - --sbin-path=/usr/nginx/nginx
+      - --conf-path=/usr/nginx/nginx.conf
+      - --pid-path=/usr/nginx/nginx.pid
+      - --error-log-path=/log/error.log
+      - --http-log-path=/log/access.log
```

Comment:
Is there a reason why all paths above are absolute ones? Shouldn't they be either relative to the snap's root filesystem or at a writeable location by the snap (e.g. the logs)?